apps/fluent-tester/src/E2E/webpack.config.ts

```diff
@@ -0,0 +1,41 @@
+import * as path from 'path';
+import { Configuration } from 'webpack';
```

Comment:
Why web pack and not metro?

Reply:
+1, we're generally moving away from webpack.

Reply:
Wasn't aware of moving away from webpack. Worked with Webpack more in the OMR, but I'll change to Metro. Nbd - thanks for letting me know

Reply:
Changed to Metro, resolving